templates/emailing/email_base.html

```diff
@@ -0,0 +1,7 @@
+<body>
+    <div dir="rtl">
+        <a href="TODO/partner/{{partner.name}}/{{order.id}}" style="font-size: large;">הזמנה</a>

```

Comment:
You realize this link won't work when you open it from the mail, right? It needs the host. Instead of `TODO`, make it a variable from the context and create something for it in settings.

Reply:
Yeah, that's why I added TODO. But this makes more sense

Reply:
Although it's a bit weird that I have to include many of the same things on every context for a staff email. But it's cleaner this way I guess...